engine/engine.go

```diff
@@ -28,6 +28,7 @@ type Engine struct {
 	Exclude          []string
 	LineSepCharacter string
 	Formatter        yamlfmt.Formatter
+	DisallowAnchors  bool
```

Comment:
Is this used anywhere? I don't think it needs to be on `Engine`.

Reply:
Done. This is now configured in the basic formatter, instead of engine-wide, right?

Reply:
Yep, so the config would look like:

```yaml
formatter:
  type: basic
  disallow_anchors: true
```